mappings/net/minecraft/client/render/VertexConsumers.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_4720 net/minecraft/client/render/VertexConsumers
+	CLASS class_4589 Union
+		FIELD field_21685 one Lnet/minecraft/class_4588;
+		FIELD field_21686 two Lnet/minecraft/class_4588;
+		METHOD <init> (Lnet/minecraft/class_4588;Lnet/minecraft/class_4588;)V
+			ARG 1 one
+			ARG 2 two
+	METHOD method_24037 union (Lnet/minecraft/class_4588;Lnet/minecraft/class_4588;)Lnet/minecraft/class_4588;
+		ARG 0 one
```

Comment:
same

Reply:
done

Reply:
Looks like it's still `one` and `two` in the parameters of `dual`